data/coffee_shops.json

```diff
@@ -242,6 +242,21 @@
         "coordinates": [-79.9666765, 40.425983]
       }
     },
+    {
+      "type": "Feature",
+      "properties": {
+        "name": "Bitty & Beau's",
+        "neighborhood": "Strip District",
+        "website": "https://www.bittyandbeauscoffee.com/location/pittsburgh",
+        "address": "1920 Smallman St Pittsburgh, PA 15222",
+        "roaster": "",
+        "photo": ""
```

Comment:
```suggestion
```

I've been meaning to fix it, but an empty `photo` property isn't supported right now. Would you mind removing it?